[docs] ETL tutorial followup (#30958)
dagster-io/dagster: docs/docs/etl-pipeline-tutorial/5-automate-your-pipeline.md

sidebar_position: 60
---

> There are several ways to automate assets [in Dagster](/guides/automate). Dagster supports both scheduled and event-driven pipelines. Here will add a schedule directly to one of our assets and make another of our assets reactive to any upstream changes. In this step you will:

Reviewer: ```suggestion
There are several ways to [automate assets ](/guides/automate) in Dagster. Dagster supports both scheduled and event-driven pipelines. Here, we will add a schedule directly to one of our assets and make another of our assets reactive to any upstream changes.
```

> - Add schedules and automation to assets.

Reviewer: ```suggestion
```

## 1. Scheduled assets

Cron-based schedules are common in data orchestration. They use time-based expressions to automatically trigger tasks at specified intervals, making them ideal for ETL pipelines that need to run consistently, such as hourly, daily, or monthly, to process and update data on a regular cadence. For our pipeline, assume that updated CSVs are uploaded at a specific time every day.

> You can define a [Schedule documentation](/guides/automate/schedules/) object in Dagster but we can also add a schedule directly to the asset with [declarative automation](/guides/automate/declarative-automation), we can include this schedule information within the <PyObject section="asset-checks" module="dagster" object="asset_check" decorator />. Now our assets will execute every day at midnight:

Reviewer: ```suggestion
While it is possible to define a standalone [schedule](/guides/automate/schedules/) object in Dagster, we can also add a schedule directly to the asset with [declarative automation](/guides/automate/declarative-automation) by including this schedule information within the <PyObject section="asset-checks" module="dagster" object="asset_check" decorator />. Now our assets will execute every day at midnight:
```
